doc/api/configuration.md

```diff
@@ -259,12 +259,16 @@ res = %sql SELECT * FROM languages
 ```{versionadded} 0.9
 ```
 
-Default: `False`
+Default: `"false"`
+
+If `"true"`, it enables named parameters `:variable`.
+If `"false"`, it disables named parameters but will parse them and return an error if they are found.
+If `"disable"`, it disables named parameters, ignores any parameters during parsing, and executes the raw query.
```

Comment:
The `ignores any parameters during parsing, and executes the raw query` is unclear. Because if we run with `disable` it still seems to be looking for binding:
```
RuntimeError: ( Named parameters are currently disabled. If you wish to use named parameters, enable them with: %config SqlMagic.named_parameters='true'. For more info, see the docs: https://jupysql.ploomber.io/en/latest/api/configuration.html#named-parameters )
(sqlite3.ProgrammingError) Incorrect number of bindings supplied. The current statement uses 1, and there are 0 supplied.
[SQL: SELECT *
FROM languages
WHERE rating > :rating]
(Background on this error at: https://sqlalche.me/e/20/f405)
If you need help solving this issue, send us a message: https://ploomber.io/community
```

Reply:
Please add an example also if possible that will demonstrate why its helpful

Reply:
I re-worded the description [here](https://github.com/ploomber/jupysql/pull/980/files#diff-7c559917b46c06e53b27fa5152bf29cb1db41f04c53f857ef5331f7a003204ffR262-R267) and added a new section [here](https://github.com/ploomber/jupysql/pull/980/files#diff-e22a552f378b0420e3ed886b3090dbec17ea676bf9a99d055349ca7eb48fbff5R297-R317) to provide more context and examples for the disable option

Reply:
I see an inconsistency here. Previously default was `False`. [Here](https://jupysql--980.org.readthedocs.build/en/980/api/configuration.html#named-parameters) it is mentioned default is `0` which disables the named parameters. But in the [tutorial](https://jupysql--980.org.readthedocs.build/en/980/user-guide/template.html#disabling-named-parameters) it is mentioned that it can be disabled using `%config SqlMagic.named_parameters = 2`. 

I think we can removed this statement: `The database may still throw an error if it itself finds malformed bind parameter syntax, but JupySql will not.`

Reply:
Apologies the description in `Configuration.md` wasn't very clear. The default is still `False`, which maps to `0`. The previous description used the word disabled but I meant that parameters are **not enabled** but still parsed. 

I've [updated the description for more clarity.](https://github.com/ploomber/jupysql/pull/980/files#diff-7c559917b46c06e53b27fa5152bf29cb1db41f04c53f857ef5331f7a003204ffR262-R266) I also removed the statement you asked.

Let me know if this description is more clear.